clubs/views.py

```diff
@@ -27,8 +27,33 @@ class ClubSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Club
-        fields = ('url', 'name', 'website', 'description', 'location',
-                  'latitude', 'longitude', 'owner', 'status', 'denial',)
+        fields = (
+            'url', # autogenerated somehow... it does not appear in the model anywhere!
```

Comment:
http://www.django-rest-framework.org/api-guide/serializers/#hyperlinkedmodelserializer

It is the PK used for this serializer.